src/Howerpoint.hs

```diff
@@ -28,30 +28,34 @@ sanitizeCode :: String -> String
 sanitizeCode = concatMap (\c -> if c == '"' then "\\\"" else [c])
 
 filterCode :: Slide -> [String]
-filterCode = catMaybes . map (\line ->
-  if isPrefixOf ">>>" line then Just $ drop 3 line
-  else if isPrefixOf "L>>" line then Just $ "let " ++ drop 3 line
-  else if isPrefixOf "H>>" line then Just $ drop 3 line
-  else Nothing)
+filterCode = catMaybes . map processLine
+  where
+    processLine :: String -> Maybe String
+    processLine line
+      | isPrefixOf ">>>" line = Just $ drop 3 line
+      | isPrefixOf "L>>" line = Just $ "let " ++ drop 3 line
+      | isPrefixOf "H>>" line = Just $ drop 3 line
+      | otherwise             = Nothing
 
 isCode :: String -> Bool
-isCode = liftA2 (||) (isPrefixOf ">>>") (isPrefixOf "L>>")
+isCode s = isPrefixOf ">>>" s || isPrefixOf "L>>" s
 
+isHiddenCode :: String -> Bool
 isHiddenCode = isPrefixOf "H>>"
 
 maxCodeLength :: Slide -> Int
-maxCodeLength = maximum . map (\line -> if isCode line then length line - 3 else 0)
+maxCodeLength = maximum . map (\line -> length line - 3) . filter isCode
 
 printCodeAndExecute :: [String] -> String
 printCodeAndExecute code = unlines (map (\x -> "putStrLn \"> " ++ (sanitizeCode x) ++ "\"") code) ++ unlines code
 
 codeFromAllPreviousSlides :: IO String
 codeFromAllPreviousSlides = undefined
 
-help = (sequence $ map (putStrLn . fst . colorize)
- ["\\k\\WTo load test presentation run \\Y:loadTestPresentation",
-  "\\k\\WTo load your own presentation run \\Y:loadPresentation <PATH-TO-YOUR-PRESENTATION>"])
-  >> pure ()
+help :: IO ()
+help = mapM_ (putStrLn . fst . colorize)
```

Comment:
I believe `mapM_` is the function you were looking for here. It's equivalent to `void . sequence . map` (where `void :: m a -> m ()` discards the return value of `m a`)